lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -75,7 +75,9 @@ def row(row)
         def eof; end
       end
 
-      NULL_CONVERSIONS = Hash.new(strict: false, proc: ->(cell_value) { cell_value }).freeze
+      # This is a Hash being initialized with a default value of a Hash, DO NOT REFORMAT TO REMOVE {}
+      # Future versions [3.4.0+] of ruby will interpret these as keywords and break.
```

Comment:
is it possible to change from "future" to just "3.4"? Future is not accurate anymore

```suggestion
      # Ruby versions from 3.4.0 onwards interpret these as keywords and break.
```

Reply:
I'll fix this up in `main` when we cut a v10. For now I'm focussing on getting the stuff out which is causing some issues.